indexer/services/comlink/src/controllers/api/v4/vault-controller.ts

```diff
@@ -0,0 +1,168 @@
+import { stats } from '@dydxprotocol-indexer/base';
+import {
+  DEFAULT_POSTGRES_OPTIONS,
+  Ordering, PaginationFromDatabase,
+  PnlTicksFromDatabase,
+  PnlTicksTable,
+  QueryableField,
+  perpetualMarketRefresher,
+  PerpetualMarketFromDatabase,
+} from '@dydxprotocol-indexer/postgres';
+import express from 'express';
+import {
+  Controller, Get, Route,
+} from 'tsoa';
+import _ from 'lodash';
+
+import { getReqRateLimiter } from '../../../caches/rate-limiters';
+import config from '../../../config';
+import { aggregatePnlTicks, handleControllerError } from '../../../lib/helpers';
+import { rateLimiterMiddleware } from '../../../lib/rate-limit';
+import ExportResponseCodeStats from '../../../request-helpers/export-response-code-stats';
+import { pnlTicksToResponseObject } from '../../../request-helpers/request-transformer';
+import { MegavaultHistoricalPnlResponse, VaultsHistoricalPnlResponse, VaultHistoricalPnl } from '../../../types';
+
+const router: express.Router = express.Router();
+const controllerName: string = 'vault-controller';
+
+// TODO(TRA-570): Placeholder interface for mapping of vault subaccounts to tickers until vaults
+// table is added.
+interface VaultMapping {
+  [subaccountId: string]: string;
+}
+
+@Route('vault/v1')
```

Comment:
You have 'v1' both here (route) and on the specific API. Could you remove one or the other?

Reply:
I can't, this is an annotation for the swagger / doc generation, the other is the actual path being registered with express.